### DISCUSSION TITLE
Grand Orgue ability to easily record within a DAW on Windows, Mac, and Linux.

### DISCUSSION THREAD

**Discussion by wpostma** [GrandOrgue/grandorgue#1134] [Help]
Perhaps thanks to the architecture of Linux audio workstations,  GrandOrgue already has an easy way (via jack or pipewire) to route audio out of the standalone grandorgue program and into  a DAW, but there does not seem to be a way to do this on Windows.

Note that JACK could be used on Windows and some apps do provide this, but I don't see a way to do this with Grand Orgue.

Perhaps a VST Plugin edition of Grand Orgue would also be a potential solution.

**Comment by oleg68** [comment]
GrandOrgue supports Jack on Windows.

**Comment by wpostma** [comment]
I have been unable to figure out how to get that to work.

**Comment by oleg68** [comment]
@wpostma 

1. Download and install a version of GrandOrgue that supports Jack for windows (all versions from GitHub do it)
2. Download and install Jack Audio Connection Kit for Windows with qjackctl session manager
3. Run qjackctl, set the jack parameters up and start the jack daemon
4. Run GrandOrgue, go to Settings->Audio, click to `Change` device, select `Jack Native` and exit with 'OK'
5. Go to `Graph` in qjackctl and make necessary connections with GrandOrgue
6. Test that everything works
7. Make a patchbay session in qjackctl for automatically connecting all applications upon their start
